Finder/TranslationFinder.php

```diff
@@ -35,13 +35,15 @@ public function __construct(KernelInterface $kernel)
      */
     public function get($domain, $locale)
     {
-        $finder = new Finder();
+        $dirToSearch = $this->getFoldersWithTranslationFiles();
```

Comment:
Why are you extracting a list of folders from the list of files and then using filesystem iteration to find files again ? Just filter the array of files directly.